releasenotes/notes/deprecate-win-system.swap-2653c98a50f42ba2.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    On Windows, the metrics `system.swap.*` are being deprecated.  They have
```

Comment:
we could be a bit more explicit by saying that they'll be removed in a future major version of the Agent, what do you think? For example:

```
    On Windows, the metrics `system.swap.*` are being deprecated and will be removed in a future major version of the Agent.
```